charts/promtail/templates/service-metrics.yaml

```diff
@@ -6,9 +6,13 @@ metadata:
   namespace: {{ include "promtail.namespaceName" . }}
   labels:
     {{- include "promtail.labels" . | nindent 4 }}
-    {{- toYaml .Values.service.labels | nindent 4 }}
+    {{- with .Values.service.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
   annotations:
```

Comment:
Include annotations into the with here.

Reply:
Good catch - helm template actually works either way but it does produce a weird yaml, better to just wrap that as well.